airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeSqlOperations.java

```diff
@@ -87,6 +90,12 @@ protected Optional<ConfigErrorException> checkForKnownConfigExceptions(Exception
           "Encountered Error with Snowflake Configuration: Current role does not have permissions on the target schema please verify your privileges",
           e));
     }
+    if (e instanceof SnowflakeSQLException && e.getMessage().contains(IP_NOT_IN_WHITE_LIST_ERR_MSG)) {
+      return Optional.of(new ConfigErrorException(
+          "Encountered Error with Snowflake Configuration: Database is not configured to access from Airbyte IPs. Please visit"
```

Comment:
AFAIK, in Snowflake IP whitelists are assigned per user rather than per database, so this should be rephrased to:
`Snowflake has blocked access from Airbyte IP address. Please make sure that your Snowflake user account's network policy allows access from all Airbyte IP addresses. See this page for the list of Airbyte IPs: https://docs.airbyte.com/cloud/getting-started-with-airbyte-cloud#allowlist-ip-addresses and this page for documentation on Snowflake network policies: https://docs.snowflake.com/en/user-guide/network-policies`

Reply:
Updated, thanks